tests/cli/keyboard_inputs.py

```diff
@@ -114,18 +174,23 @@ def write_to_socket():
     sock = connect_tcp()
     while(True):
         mutex.acquire()
-        string_to_send = ''.join(bits)
-        print("sending", string_to_send.encode())
-        sock.send(string_to_send.encode()) # send the 'bitstring' over tcp using socket object
+        gamepad_to_send = ''.join(gamepad_bits)
+        print("sending gamepad", gamepad_to_send.encode())
+        sock.send(gamepad_to_send.encode()) # send the 'bitstring' over tcp using socket object
+        keyboard_to_send = ''.join(keyboard_bits)
+        print("sending keyboard", keyboard_to_send.encode())
+        sock.send(keyboard_to_send.encode())
```

Comment:
Think it would be easier to send over a 64-bit integer rather than a bit string. But maybe this is fine for now since it works

Reply:
Can use `ctypes` to get 64bit int. See https://stackoverflow.com/questions/19716315/how-to-create-a-fixed-size-unsigned-integer-in-python

Reply:
Great suggestion! I'll make a separate issue that can be tackled on a different branch